library/kong

```diff
@@ -1,13 +1,25 @@
 Maintainers: Kong Docker Maintainers <team-gateway@konghq.com> (@team-gateway-bot)
 GitRepo: https://github.com/Kong/docker-kong.git
 
-Tags: 3.0.0-alpine, 3.0.0, 3.0, alpine, latest
+Tags: 3.0.1-alpine, 3.0.1, 3.0, alpine, latest
+GitCommit: b2057cd3432726c2122873a3740e31a68a03cecd
+GitFetch: refs/tags/3.0.1
+File: Dockerfile.apk
+Architectures: amd64, arm64v8
+
+Tags: 3.0.1-ubuntu, 3.0-ubuntu, ubuntu
+GitCommit: b2057cd3432726c2122873a3740e31a68a03cecd
+GitFetch: refs/tags/3.0.1
+Directory: ubuntu
+Architectures: amd64, arm64v8
+
+Tags: 3.0.0-alpine, 3.0.0
 GitCommit: 5a2610d900717a4a6c8ee98bc421d0d4599b126f
 GitFetch: refs/tags/3.0.0
 Directory: alpine
 Architectures: amd64, arm64v8
 
-Tags: 3.0.0-ubuntu, 3.0-ubuntu, ubuntu
+Tags: 3.0.0-ubuntu
```

Comment:
Any particular reason to keep the `3.0.0` versions listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
No probably not :thinking: